src/main/scala/org/broadinstitute/dsde/workbench/sam/config/package.scala

```diff
@@ -117,7 +121,8 @@ package object config {
       config.getInt("groupSync.workerCount"),
       config.getString("notifications.topicName"),
       config.as[GoogleKeyCacheConfig]("googleKeyCache"),
-      config.as[Option[String]]("resourceNamePrefix")
+      config.as[Option[String]]("resourceNamePrefix"),
+      config.as[Option[Seq[ServiceAccountConfig]]]("adminSdkServiceAccounts").getOrElse(Seq.empty)
```

Comment:
Can we do `config.as[Option[NonEmptyList[ServiceAccountConfig]]]` since `Seq` doesn't guarantee it's not empty. So `Option[Seq[A]]` doesn't really make much sense